### ISSUE TITLE
SRW - exported script with tabulated undulator does not work

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#511]
Found by @ochubar:
> The exported Python script with the tabulated undulator does not work.

I was able to reproduce the issue. The reason was that there is no information about the magnetic measurements sub-folder (`und_mdir`) and the magnetic measurements for different gaps summary file (`und_mfs`) passed to the exported file. Need to fix this issue, so that the exported file would look like:
```python
    ['und_mdir', 's', 'magn_meas_epu105_test_qp_bms', 'name of magnetic measurements sub-folder'],
    ['und_mfs', 's', 'epu105_esm_meas_qp_test_sum_BMs.txt', 'name of magnetic measurements for different gaps summary file'],
```

**Comment by mrakitin** [comment]
This ticket is related to #243.

**Comment by mrakitin** [comment]
Should be fixed within #243.
